Assets/UnZENity-Core/Scripts/Globals/GameData.cs

```diff
@@ -39,6 +39,9 @@ public static class Dialogs
             public static List<InfoInstance> Instances = new();
             public static bool IsInDialog;
 
+            // I'm too stupid to make it work through vfs need someone to help here
+            public static CutsceneLibrary CutsceneLibrary = new($"{GameGlobals.Settings.Gothic1Path}\\_work\\DATA\\scripts\\Content\\CUTSCENE\\Ou.csl");
```

Comment:
The lookup is fine this way. Just put it inside GuzBootstrapper please. We already have a function where we load all VFS and (e.g.) the GothicVM. Just add another method like LoadCutscenes (Or better LoadSubtitles? ;-) ).

Reply:
ZenKit goes with calling it Cutscenes but I agree that we can call it LoadSubtitles here but keep the current naming in deeper levels of scope. 🫡